configurations/default/settings.yml

```diff
@@ -0,0 +1,42 @@
+entries:
+  - client/index.js:assets/index.js
+  - client/index.css:assets/index.css
+map:
```

Comment:
Were you thinking this kind of data could be put into a user store and a user table in the database?